docs/developer-docs/functionality/bitcoin/local-development.md

```diff
@@ -0,0 +1,292 @@
+        ...
+        "3e0410961a3b144d16dcdc95d73c262e0ec09e9812ce33835b2a55606d2be84b"
+    ]
+
+In the logs of `dfx`, you should see something like this within a few seconds:
+
+    INFO .../blockchainmanager Added headers: Height = 101, Active chain's tip = 3e0410961a3b144d16dcdc95d73c262e0ec09e9812ce33835b2a55606d2be84b
+    DEBG .../ic_btc_canister/heartbeat New Bitcoin tip height: 101
+
+These logs indicate that your local `dfx` project has ingested the 100 new blocks you just mined.
+The BTC your canister owns can now be spent.
+
+## Sending Bitcoin
+
+You can send Bitcoin using the `send` endpoint on your canister.
+
+In the Candid UI, add a destination address and an amount to send. In the example
+below, we're sending 1 BTC to the address `n2dcQfuwFw7M2UYzLfM6P7DwewsQaygb8S`.
+
+![Sending Bitcoin Transaction](../_attachments/bitcoin-local-send-tx.png)
+
+Via command line, the same call would look like this:
+
+    dfx canister call basic_bitcoin send '(record { destination_address = "n2dcQfuwFw7M2UYzLfM6P7DwewsQaygb8S"; amount_in_satoshi = 100000000; })'
+
+The command above creates a transaction and sends it out to your local Bitcoin node.
+For more details on how the `send` endpoint works, see the [Deploying Your First Bitcoin Dapp](./deploying-your-first-bitcoin-dapp#sending-bitcoin) tutorial.
+
+There still remains one additional step, which is to mine a block so that the transaction
+you just sent becomes part of the blockchain. Run the following from the `bitcoind` directory:
+
+    ./bin/bitcoin-cli -conf=$(pwd)/bitcoin.conf generatetoaddress 1 mtbZzVBwLnDmhH4pE9QynWAgh6H3aC1E6M
+
+The above command is similar to what we did in the section on [Coinbase Maturity](./local-development#coinbase-maturity).
+And, similarly, you should see the logs in `dfx` updating to indicate the ingestion of this new
+block.
+
+If everything worked, you should now see that address `n2dcQfuwFw7M2UYzLfM6P7DwewsQaygb8S`
+has a balance of 1 BTC.
+
+![Checking Bitcoin Balance](../_attachments/bitcoin-local-balance-after-send.png)
+
+## Troubleshooting
+
+### Sending Transactions
+
+If you're trying to send a transaction and the transaction isn't being mined,
+try sending the same transaction via `bitcoin-cli`, as it can reveal helpful errors:
+
+    ./bin/bitcoin-cli -conf=$(pwd)/bitcoin.conf sendrawtransaction <tx-in-hex>
+
+For example:
+
+    ./bin/bitcoin-cli -conf=$(pwd)/bitcoin.conf sendrawtransaction 0200000001b0ca9600da1057765dab692467579cb309aba2524d0cd45376874d6e39e1cd50000000006a47304402200024c768daeb38591438cb3cc7e40212442e8a3662f7465bd466d00ceede8014022069ca804f7ca0ba757c9531c5c37c41943a1a1b53b96697b27e928fb35637c5b10121024f9671d4f1a434cfa6c5ca863670b34f2836c259ef2c8c33f4195f2997dc3ee3ffffffff0200e1f505000000001976a914ce0966271055a5b17abb0e9021a7eafdfe557d3088ac1f101024010000001976a9143fc55ebf65a72a3658f0e14c75c99f6eae65b9b388ac00000000
+    error code: -25
+    error message:
+    bad-txns-inputs-missingorspent
+
+### Resetting the state
```

Comment:
In my opinion this section should contain a warning as we are inviting the canister developer to erase in a permanent way its `.dfx` that may contain far more things than just the Bitcoin state synchronized by dfx.

Reply:
I agree. Done.